plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlValidatorTest.java

```diff
@@ -39,27 +38,30 @@
 
 public class WebXmlValidatorTest {
```

Comment:
It seems to me that these tests are duplicate and should rather be in the `WebXmlScannerTest` class and test `WebXmlScanner`. By looking at the `WebXmlValidator` class the tests in `WebXmlValidatorTest` should exercise the code in that class without any real validation, i.e. mock out the IResource and WebXmlSaxParser instances.